### ISSUE TITLE
Launch app with application argument -FIRAnalyticsDebugEnabled

### ISSUE THREAD

**Issue by jeremypele** [EddyVerbruggen/nativescript-plugin-firebase#427]
Hi,

I'm integrating the the plugin for an ios version and I can't find a way to launch the app with `tns run ios` and specifying that I want Analytics debug mode activated (with `-FIRAnalyticsDebugEnabled`)

I've launched the app with the scheme option on XCode, it works well but it can't refresh the app like `tns run` does.

Any idea how to do that?

**Comment by johnnyzen** [comment]
I would love to get debug mode working on analytics, so I can test events live

**Comment by NathanWalker** [comment]
I'm also curious on this @EddyVerbruggen is this as simple as adding:
```
FIRAnalyticsDebugEnabled = true
```
in the build.xcconfig?

**Comment by EddyVerbruggen** [comment]
Does that work?

**Comment by NathanWalker** [comment]
Looks like *not* unfortunately :(

**Comment by jeremypele** [comment]
Yeah in xcode you have to define the option in a `scheme`, not in the `build.xcconfig` :/

**Comment by sulemanAbrar1996** [comment]
Hey guys, did you find any solution about this?

**Comment by sulemanAbrar1996** [comment]
@jeremypele  @EddyVerbruggen @NathanWalker @johnnyzen  i my case app builds succefully but when runs then it shows error 
 6.13.0 - [Firebase/Analytics][I-ACS023008] To enable debug logging set the following application argument: -FIRAnalyticsDebugEnabled
